trickster/endpoints/internal.py

```diff
@@ -47,8 +47,8 @@ def create_route(route: InputRoute, mocked_router: Router = Depends(get_router))
         new_route.validate_existing_response_validator_combinations()
         mocked_router.add_route(new_route)
         return new_route
-    except ValidationError as e:
-        raise HTTPException(status_code=400, detail=f'Failed validation: {e}') from e
+    except pydantic.ValidationError as exception:
+        raise ValidationError(str(exception))
```

Comment:
It's usually good practice to do `raise ValidationError(str(exception)) from exception`. When debugging, you can then see what caused it